third_party/move/move-compiler-v2/src/ast_simplifier.rs

```diff
@@ -1001,16 +1001,12 @@ impl<'env> ExpRewriterFunctions for SimplifierRewriter<'env> {
             return Some(body.clone());
```

Comment:
> This required some changes to pass ordering to do unused assignment after inlining but before simplifier.

Unused assignment checking is done on the stackless bytecode, can we reorder it to be placed after inlining and before simplifier which both happen on the AST?

Reply:
I misspoke here, I was talking about "unused variable" but I was also wrong about needing to re-order.  No re-ordering now, everything looks good.